data/sql/updates/pending_db_world/rev_1654272234519507300.sql

```diff
@@ -0,0 +1,534 @@
+(9429, 0, 'zhCN', '我来做示范！把练习用的降落伞包递给我！'),
+(9470, 0, 'zhCN', '无头骑士是谁？'),
+(9586, 0, 'zhCN', '旧斯坦索姆……为什么送我回到这一时间流？'),
+(9594, 0, 'zhCN', '是什么决定？'),
+(9595, 0, 'zhCN', '永恒之龙打算如何破坏这一时间流呢？'),
+(9610, 0, 'zhCN', '他们想干什么？'),
+(9611, 0, 'zhCN', '什么？加入王子的军队？'),
+(9612, 0, 'zhCN', '好吧，克罗米。'),
+(9724, 0, 'zhCN', '能给我一颗橙子吗？'),
+(9724, 2, 'zhCN', '我可能需要一颗木瓜。'),
+(9761, 0, 'zhCN', '我需要水下呼吸效果，盖伦。'),
+(9823, 1, 'zhCN', '我把禁魔监狱的钥匙弄丢了。'),
+(9823, 2, 'zhCN', '我把破碎大厅的钥匙弄丢了。'),
+(9823, 3, 'zhCN', '我把灼热峡谷的钥匙弄丢了。'),
+(9823, 5, 'zhCN', '我把哈拉迈德之眼弄丢了。'),
+(9823, 6, 'zhCN', '我把卡拉赞的钥匙弄丢了。'),
+(9823, 7, 'zhCN', '我丢掉了紫罗兰要塞的钥匙。'),
+(9823, 8, 'zhCN', '我遗失了灌注精华的月亮石。你能再给我一个吗？'),
+(9855, 0, 'zhCN', '我还有几个问题，盖米尔。'),
+(9860, 0, 'zhCN', '我还有几个问题，盖米尔。'),
+(9891, 1, 'zhCN', '我要参加对抗巨龙的战斗。能给我一只雄鹰吗，弗亚林？'),
+(9906, 0, 'zhCN', '很抱歉打扰到你，酋长。但是，你的兄弟的灵魂可能遇到了危险。你能把关于他的回忆告诉我吗？'),
+(9917, 1, 'zhCN', '我准备好进入风暴峭壁了。'),
+(10040, 3, 'zhCN', '我能帮你什么忙吗？'),
+(10160, 0, 'zhCN', '联盟区'),
+(10160, 1, 'zhCN', '部落区'),
+(10273, 0, 'zhCN', '拎起冰霜豹的尾巴，检查一下它是雄性还是雌性。'),
+(10274, 0, 'zhCN', '拎起冰爪熊的尾巴，检查一下它是雄性还是雌性。'),
+(10314, 0, 'zhCN', '我们这就来帮忙！'),
+(10316, 1, 'zhCN', '乐意为你效劳，女士。听说你曾是一把传说之剑的守护者。你能告诉我该去哪儿寻找这把剑吗？'),
+(10317, 0, 'zhCN', '前往一家银行。'),
+(10317, 1, 'zhCN', '拜访一名商人。'),
+(10317, 2, 'zhCN', '访问邮箱。'),
+(10317, 3, 'zhCN', '暗矛冠军的旗帜'),
+(10317, 4, 'zhCN', '被遗忘者冠军的旗帜'),
+(10317, 5, 'zhCN', '奥格瑞玛冠军的旗帜'),
+(10317, 6, 'zhCN', '银月城冠军的旗帜'),
+(10317, 7, 'zhCN', '雷霆崖冠军的旗帜'),
+(10318, 0, 'zhCN', '前往一家银行。'),
+(10318, 1, 'zhCN', '拜访一名商人。'),
+(10318, 2, 'zhCN', '访问邮箱。'),
+(10318, 3, 'zhCN', '达纳苏斯冠军的旗帜'),
+(10318, 4, 'zhCN', '埃索达冠军的旗帜'),
+(10318, 5, 'zhCN', '诺莫瑞根冠军的旗帜'),
+(10318, 6, 'zhCN', '铁炉堡冠军的旗帜'),
+(10318, 7, 'zhCN', '暴风城冠军的旗帜'),
+(10389, 2, 'zhCN', '传送到巨人熔炉。'),
+(10389, 4, 'zhCN', '传送到奥杜尔的前厅。'),
+(10389, 5, 'zhCN', '传送到破碎通道。'),
+(10389, 6, 'zhCN', '传送到生命温室。'),
+(10389, 7, 'zhCN', '传送到发明大厅。'),
+(10389, 8, 'zhCN', '传送到疯狂阶梯。'),
+(10599, 1, 'zhCN', '我们要向这些野兽再发出狠狠的一击！'),
+(10600, 1, 'zhCN', '我们已经准备好了。这一次情况就不一样了。'),
+(10609, 1, 'zhCN', '我们已经准备好了再次同那名法师作战。'),
+(10610, 0, 'zhCN', '我已经为下一个挑战做好了准备。'),
+(10610, 1, 'zhCN', '开始。'),
+(10638, 1, 'zhCN', '我希望能再次开始获取经验。'),
+(10668, 0, 'zhCN', '北风苔原'),
+(10668, 1, 'zhCN', '嚎风峡湾'),
+(10668, 2, 'zhCN', '索拉查盆地'),
+(10668, 3, 'zhCN', '冰冠冰川'),
+(10668, 4, 'zhCN', '风暴峭壁'),
+(10668, 5, 'zhCN', '地下……'),
+(10678, 1, 'zhCN', '等到我们将他们彻底消灭的时候，你会更吃惊的！'),
+(10679, 1, 'zhCN', '我们必须提高我们的团队协作能力，才能战胜那两个家伙。'),
+(10687, 1, 'zhCN', '我们已经做好了一切准备！'),
+(10688, 1, 'zhCN', '尽管把她们带出来，然后好好看着吧。'),
+(10692, 0, 'zhCN', '感谢你的赞扬，演武场统领。'),
+(10693, 0, 'zhCN', '这真奇怪……'),
+(10704, 0, 'zhCN', '我完成了十字军的试炼。'),
+(10704, 1, 'zhCN', '我还没有完成十字军的试炼。'),
+(10704, 2, 'zhCN', '我们要向这些野兽再发出狠狠的一击！'),
+(10706, 0, 'zhCN', '是的，我们已经准备好了迎接即将到来的挑战。'),
+(10847, 0, 'zhCN', '开始战斗'),
+(10854, 0, 'zhCN', '奥术师泰拜林说你会借我一件战袍。'),
+(10854, 1, 'zhCN', '魔导师哈瑟尔说你会借我一件战袍。'),
+(10933, 0, 'zhCN', '我们准备好了，穆拉丁。'),
+(10934, 0, 'zhCN', '确信无疑。出发吧！'),
+(10943, 0, 'zhCN', '有什么我能效劳的，我的女士？'),
+(10952, 0, 'zhCN', 'Lok''tar ogar！我们准备好了！兽人兄弟们，冲啊！'),

```

Comment:
this line is erroring out the check with the ''
```suggestion
(10952, 0, 'zhCN', 'Lok tar ogar！我们准备好了！兽人兄弟们，冲啊！'),
```